lstchain/scripts/longterm_dl1_check.py

```diff
@@ -487,22 +490,17 @@ class pixwise_info(tables.IsDescription):
 
     # We write out the camera geometry information, assuming it is the same
     # for all files (hence we take it from the first one):
-    cam_description_table = \
-        Table.read(files[0], path='instrument/telescope/camera/LSTCam')
-    geom = CameraGeometry.from_table(cam_description_table)
-    geom.to_table().write(output_file_name,
-                          path=f'/instrument/telescope/camera/LSTCam',
-                          append=True, serialize_meta=True)
+    subarray_info = SubarrayDescription.from_hdf(files[0])
+    subarray_info.to_hdf(output_file_name)
 
     plot(output_file_name)
 
 
 def plot(filename='longterm_dl1_check.h5'):
 
     # First read in the camera geometry:
-    cam_description_table = \
-        Table.read(filename, path='instrument/telescope/camera/LSTCam')
-    camgeom = CameraGeometry.from_table(cam_description_table)
+    subarray_info = SubarrayDescription.from_hdf(filename)
+    camgeom = subarray_info.camera_types[0].geometry
```

Comment:
I see a problem here when having more than one camera type. Here it is just the only one present in the file, but what happens with mc arrays with more telescopes?

Reply:
Shouldn't you read the geometry of the telescope directly (from its `tel_id`)?
Something such as
```
subarray_info.tel[tel_id].camera.geometry
```

Reply:
Yes, `subarray.tel[tel_id].camera.geometry`

Reply:
Which is the best way of getting `tel_id`? Can it be fetched from the same subarray info, config file?

Reply:
I tried to do it this way. @vuillaut @maxnoe please check that I did it properly. I set `tel_id` by default to 1 either via command line or as an argument of the corresponding function.